src/bap_mc/bap_mc.ml

```diff
@@ -140,6 +158,19 @@ module Cmdline = struct
     let doc = "Output instruction kinds." in
     Arg.(value & flag & info ["show-kinds"] ~doc)
 
+  let show_oplen =
+    let formats = [
+	"hex", `hex;
+	"dec", `dec;
+      ] in
+    let doc = sprintf
+		"Output recognized opcode length, optional \
```

Comment:
this is not opcode length, this is length of the overall instruction including operands. And, are you sure that we really need to different formats to output length? As it clobber the interface, without really buying to much functionality?